src/integrations/altervistaAPI.ts

```diff
@@ -0,0 +1,53 @@
+import type{ PartOfSpeech, Synonym, SynonymProvider } from "src/integrations/types";
+
+export class AltervistaSynonymProvider implements SynonymProvider {
+    name = "Altervista";
+    url = "http://thesaurus.altervista.org/";
+    //Look up more later
+    supportedLanguages: string[] = [
+        "es",
+        "it",
+        "fr",
+        "de",
+    ];
+
+    languageCodes = {
+        "es": "es_ES",
+        "it": "it_IT",
+        "fr": "fr_FR",
+        "de": "de_DE",
+    }
+
+    //This is limited to 5000 queries/day
+    TOKEN = "P4QAmqYIN1DY6XjlQJht"
```

Comment:
This is super problematic. Any plan to solve this?

Reply:
I could add "Backup" Tokens, but that would probably against TOS. I of course know that it is hard to create free API's because they need to pay for their servers too, but I also don't have the money to pay for higher licenses for a free Plugin.

Did we hit any limits yet?

Reply:
![image](https://user-images.githubusercontent.com/59741989/122563683-c5cf9000-d044-11eb-8070-88f9ae52d091.png)
Looks like we are still far away from the Limit. This is also all the Data i can see with the Key.